SPECS/glib/glib.spec

```diff
@@ -55,11 +57,13 @@ Gsettings schemas compiling tool
 %setup -q
 %patch0 -p1
 %patch1 -p1
+%patch2 -p1
```

Comment:
Suggestion: switch to `%autosetup -p1` instead.

Reply:
Omitting this change since it's already successfully built as-is, and I'd rather not make a change that requires a rebuild/retest right before I go on vacation